tasks/linux.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+action=$PT_action
+name=$PT_name
+command_exists=0
+service_managers[0]="systemctl"
+service_managers[1]="service"
+service_managers[2]="initctl"
+
+# example cli /opt/puppetlabs/puppet/bin/bolt  task run service::linux action=stop name=ntp --nodes localhost --modulepath /etc/puppetlabs/code/modules --password puppet --user root
+
+check_command_exists() {
+  (which $1) > /dev/null 2>&1
```

Comment:
Should we use 'command -v' for POSIX compliance?

Should $1 be double quoted to prevent globbing?

Reply:
the script is explicitly calling for bash, so posix compliance is not an issue.

yes to the double quoting.